modules/image-loading-optimization/module.php

```diff
@@ -336,4 +342,38 @@ private function high_priority_element_flag( $value = null ) {
 		}
 		return $high_priority_element;
 	}
+
+	/**
+	 * Enqueue the module editor scripts.
+	 *
+	 * @return void
+	 */
+	public function enqueue_editor_scripts() {
+		wp_enqueue_script(
+			$this->get_name() . '-editor',
+			$this->get_js_assets_url( $this->get_name() . '-editor' ),
+			[ 'elementor-common', 'elementor-editor-modules' ],
+			ELEMENTOR_VERSION,
+			array(
+				'strategy'  => 'defer',
+			)
+		);
+	}
+
+	/**
+	 * Enqueue the module preview scripts.
+	 *
+	 * @return void
+	 */
+	public function enqueue_preview_scripts() {
+		wp_enqueue_script(
+			$this->get_name() . '-preview',
+			$this->get_js_assets_url( $this->get_name() . '-preview' ),
+			[],
+			ELEMENTOR_VERSION,
+			array(
```

Comment:
Since elementor would also support older version of WordPress we need to have a version check for `>=6.3` here and only then have the strategy.